src/librustc_target/spec/asmjs_unknown_emscripten.rs

```diff
@@ -42,7 +42,6 @@ pub fn target() -> Result<Target, String> {
         target_vendor: "unknown".to_string(),
         data_layout: "e-p:32:32-i64:64-v128:32:128-n32-S128".to_string(),
         arch: "asmjs".to_string(),
-        linker_flavor: LinkerFlavor::Em,
```

Comment:
I think this ay break emscripten, right? I think there's no other way to say that "the emscripten target uses emcc" by default, right?